pmm-app/src/pmm-update/components/InfoBox/InfoBox.tsx

```diff
@@ -13,27 +23,47 @@ export const InfoBox: FC<InfoBoxProps> = ({
 }) => {
   const styles = useStyles(getStyles);
 
-  return (
-    <section data-qa="updates-info" className={styles.infoBox}>
-      {hasNoAccess ? (
+  if (hasNoAccess) {
```

Comment:
I don't see a clear advantage of having it this way besides the fact that we don't need to turn off the rule. This way it needs more code and multiple returns (which might be a bad practice as well). Anyway either one is fine, this is just a minor detail, so let's go!

Reply:
Arguably a bad practice, the code is much easier to read and therefore to maintain.

Reply:
We do not want to save on the lines of code, we do want to be able to understand how it works. And the nested ternary fails short on that end.

Reply:
If a particular eslint rule is bad or does more harm than good, I suggest we convince the community. Then it will be objective.